oemof/solph/components.py

```diff
@@ -199,6 +207,13 @@ def _check_invest_attributes(self):
             e2 = ("Overdetermined. Three investment object will be coupled"
                   "with three constraints. Set one invest relation to 'None'.")
             raise AttributeError(e2)
+        if (self.investment and
+                self.fixed_losses_absolute != solph_sequence(0) and
+                self.investment.existing == 0 and
+                self.investment.minimum == 0):
+            e3 = ("With fixed_losses_absolute > 0, either investment.existing "
```

Comment:
This error is not tested, but I do not know who forgot it. We should add the test.